www/js/controllers.js

```diff
@@ -59,6 +59,14 @@ angular.module('starter.controllers', ['starter.services'])
                         });
       }
     });
+    var vehicles = $resource('http://bustracker.pvta.com/infopoint/rest/vehicles/getallvehicles').query({}, function(){
```

Comment:
As we start looking at breaking up these files a bit, the non-changing bit of this url would be good to put in a config file somewhere.